cmd/run.go

```diff
@@ -66,20 +70,31 @@ func run(sections []string, benchmarkVersion string) (err error) {
 	return nil
 }
 
-func getTestYamlFiles(sections []string, benchmarkVersion string) (yamlFiles []string, err error) {
+func providesTargets(benchmarkVersion string) bool {
```

Comment:
Not sure about this function name, looks like it is getting the benchmark version number rather than checking targets? 

I was going to say that I don’t see where this is checking the targets, but I suspect you are thinking that there aren’t targets for CIS < 1.5.

All versions do provide some targets - before 1.5 the targets were master and node, and now there are some additional ones. (Otherwise you’d only be able to use the —targets option with >=1.5 which seems an unnecessary restriction)